.gitignore

```diff
@@ -2,3 +2,4 @@ target
 optimized_wasm
 dist
 .DS_Store
+noosphere.h
```

Comment:
I wouldn't mind vendoring the header file once a ffi helper (safer-ffi, cbindgen, uniffi) if at least to link to the C API

Reply:
We should make it trivial to generate it locally. I'm not opposed to checking it in, but it really should never be modified by hand.

Reply:
+1